pkg/combustion/templates/13-embedded-registry.sh.tpl

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -euo pipefail
+
+mkdir /opt/hauler
+
+mv {{ .EmbeddedRegistryTar }} /opt/hauler/{{ .EmbeddedRegistryTar }}
+mv hauler /usr/bin/hauler
```

Comment:
@rdoxenham is it better to have this under `/usr/bin` or `/usr/local/bin`?

Reply:
`/usr/local/bin` is better here as it's excluded from rollbacks, i.e. it's part of the persistent/mutable parts of the OS. We may need to mount it via combustion though, so we should test this.